flaml/model.py

```diff
@@ -921,7 +921,13 @@ def search_space(cls, data_size, task, **params):
 
 
 class SKLearnEstimator(BaseEstimator):
-    """The base class for tuning scikit-learn estimators."""
+    """
+    The base class for tuning scikit-learn estimators.
+
+    Subclasses should modify the function signature of ``__init__`` to
+    ignore the values in ``config`` that are not relevant to the constructor
+    of their estimator.

```

Comment:
```suggestion
    Subclasses can modify the function signature of ``__init__`` to
    ignore the values in ``config`` that are not relevant to the constructor
    of their underlying estimator. For example, some regressors in ``scikit-learn``
    don't deal with the ``n_jobs`` parameter contained in ``config``. For those,
    one can add ``n_jobs=1,`` before ``**config`` to make sure ``config`` doesn't
    contain ``n_jobs``.
```
Is this more clear? I don't know the difference between ` and `` in docstr. Please make sure it looks good in the website.

Reply:
Just update. I did check and it does look good 😊